valhalla/userrequests/models.py

```diff
@@ -175,6 +178,10 @@ def blocks(self):
             logger.error('Could not connect to the pond.')
             return BlockSerializer([], many=True).data
 
+    @classmethod
+    def need_accounting(clazz):
+        return clazz.objects.filter(state__in=['COMPLETED', 'CANCELED', 'WINDOW_EXPIRED'], accounted=False)
```

Comment:
we have a TERMINAL_STATES list in state_changes.py, maybe we should pull that into the class and reuse it here.

Reply:
Good idea.